ingest-app/docs/api.md

```diff
@@ -159,8 +159,10 @@ The following fields are validated:
 * [Data Format](https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/dataformat?format=csv) - Archival and Distribution File Format, and GetData Format
 * [ProcessingLevel] (https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/productlevelid?format=csv) - productlevelid
 
-**Note**: cmr-validate-keywords is set to TRUE by default
+**Note**: if cmr-validate-keywords header is not set explicitly, it will behave as if it was set to FALSE by default behind the scenes
```

Comment:
Since this line probably won't be updated again until the feature toggle is removed, I'm wondering if we need to communicate here that work is being done to change it to default true in the various environments. Perhaps a note in parens?
Though I'm not sure how important that is or how quick we'll be getting that work done

Reply:
I think we can leave it and I will make an update to this doc as soon as we toggle this feature on in PROD